support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierFirstStepLanding.tsx

```diff
@@ -0,0 +1,163 @@
+import { Header } from 'components/headers/simpleHeader/simpleHeader';
+import { PageScaffold } from 'components/page/pageScaffold';
+import { useOtherAmountValidation } from 'helpers/customHooks/useFormValidation';
+import {
+	AUDCountries,
+	Canada,
+	EURCountries,
+	GBPCountries,
+	International,
+	NZDCountries,
+	UnitedStates,
+} from 'helpers/internationalisation/countryGroup';
+import { resetValidation } from 'helpers/redux/checkout/checkoutActions';
+import { getContributionType } from 'helpers/redux/checkout/product/selectors/productType';
+import { getUserSelectedAmount } from 'helpers/redux/checkout/product/selectors/selectedAmount';
+import {
+	useContributionsDispatch,
+	useContributionsSelector,
+} from 'helpers/redux/storeHooks';
+import { navigateWithPageView } from 'helpers/tracking/ophan';
+
+const checkoutBtnStyleOverrides = css`
+	width: 100%;
+	justify-content: center;
+`;
+
+const darkBackgroundContainerMobile = css`
+	display: flex;
+	background: linear-gradient(
+		to bottom,
+		${palette.brand[400]} 441px,
+		${palette.neutral[97]} 441px,
+		${palette.neutral[97]} 100%
+	);
+	${until.tablet} {
+		background-color: ${palette.brand[400]};
+		border-bottom: 1px solid ${palette.brand[600]};
+	}
+	min-height: 480px;
+	${from.desktop} {
+		min-height: 440px;
+	}
+	padding-top: ${space[6]}px;
+`;
+
+const links = [
+	{
+		href: 'https://www.theguardian.com/info/privacy',
+		text: 'Privacy policy',
+		isExternal: true,
+	},
+	{
+		text: 'Privacy settings',
+		onClick: () => {
+			cmp.showPrivacyManager();
+		},
+	},
+	{
+		href: 'https://www.theguardian.com/help/contact-us',
+		text: 'Contact us',
+		isExternal: true,
+	},
+	{
+		href: 'https://www.theguardian.com/help',
+		text: 'Help centre',
+		isExternal: true,
+	},
+];
+
+export function ThreeTierLandingPage(): JSX.Element {
+	const dispatch = useContributionsDispatch();
+	const navigate = useNavigate();
+	const contributionType = useContributionsSelector(getContributionType);
+	const amount = useContributionsSelector(getUserSelectedAmount);
+
+	const { abParticipations } = useContributionsSelector(
+		(state) => state.common,
+	);
+
+	const { countryGroupId } = useContributionsSelector(
+		(state) => state.common.internationalisation,
+	);
+
+	const countrySwitcherProps: CountryGroupSwitcherProps = {
+		countryGroupIds: [
+			GBPCountries,
+			UnitedStates,
+			AUDCountries,
+			EURCountries,
+			NZDCountries,
+			Canada,
+			International,
+		],
+		selectedCountryGroup: countryGroupId,
+		subPath: '/contribute',
+	};
+
+	const proceedToNextStep = useOtherAmountValidation(() => {
+		const destination = `checkout?selected-amount=${amount}&selected-contribution-type=${contributionType.toLowerCase()}`;
+		navigateWithPageView(navigate, destination, abParticipations);
+	}, false);
+
+	useEffect(() => {
+		dispatch(resetValidation());
+	}, []);
+
+	return (
+		<PageScaffold
+			id="supporter-plus-landing"
```

Comment:
```suggestion
			id="supporter-plus-landing-three-tier"
```
Maybe something to identify three tier supporter plus page?